CHANGES.rst

```diff
@@ -84,6 +84,33 @@ Enhancements
 API changes
 -----------
 
+*  of the :py:meth:`~.model.BaseModel.fit` and
+  :py:meth:`~.model.BaseModel.multifit`:
+  
+  * The ``fitter`` keyword has been renamed to ``optimizer``.
+  * The values that the ``optimizer`` keyword take have been renamed
+    for consistency with scipy:
+
+    * "fmin" -> "Nelder-Mead",
+    * "fmin_cg" -> "CG",
+    * "fmin_ncg" -> "Newton-CG",
+    * "fmin_bfgs" -> "BFGS",
+    * "fmin_l_bfgs_b" -> "L-BFGS-B",
+    * "fmin_tnc" -> "TNC",
+    * "fmin_powell" -> "Powell",
+    * "mpfit" -> "lm" (in combination with `"bounded=True"`),
+    * "leastsq" -> "lm",
+    * Passing integer arguments to ``parallel`` to select the number of
+      workers is now deprecated. Use ``parallel=True, max_workers={value}``
+      instead.
+* The ``method`` keyword has been renamed to ``loss_function``.
+* The ``loss_funtion`` value ``"ml"`` has been renamed to ``ml-poisson``.
```

Comment:
Typo in loss function

Reply:
Fixed, thanks. Could you have a look at it again? I should be fairly complete now.

Reply:
Have done so

Reply:
Thanks!

What do you think about renaming:

* "ml-poisson" -> "ML-Poisson" or "ML-poisson"?
* "lm" -> "LM"

to make it clearer that they're acronyms?

Reply:
- I think keeping `"lm"` lowercase is important to match `scipy.optimize.least_squares`.
- Happy with `"ML-poisson"`